pkg/iac/terraform/state/versions.go

```diff
@@ -0,0 +1,43 @@
+package state
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/go-version"
+)
+
+var (
+	// UnsupportedVersionConstraints is an array of version constraints known to be unsupported.
+	// If a given state matches one of these, all resources of the related state will be ignored and marked as drifted.
+	UnsupportedVersionConstraints = []string{"<0.11.0"}
+)
+
+type UnsupportedVersionError struct {
+	StateFile string
+	Version   *version.Version
+}
+
+func (u *UnsupportedVersionError) Error() string {
+	return fmt.Sprintf("%s was generated using Terraform %s which is currently not supported by driftctl\n"+
+		"Please read documentation at https://docs.driftctl.com/0.7.0/limitations", u.StateFile, u.Version)
```

Comment:
We're gonna switch the link to the deep link one `https://docs.driftctl.com/limitations`